airflow/executors/celery_executor.py

```diff
@@ -166,6 +167,31 @@ def send_task_to_executor(task_tuple: TaskInstanceInCelery) \
     return key, command, result
 
 
+# pylint: disable=unused-import
+@celery_import_modules.connect
+def on_celery_import_modules(*args, **kwargs):
+    """
+    Preload some "expensive" airflow modules so that every task process doesn't have to import it again and
+    again.
+
+    Loading these for each task adds 0.3-0.5s *per task* before the task can run. For long running tasks this
+    doesn't matter, but for short tasks this starts to be a noticeable impact.
+    """
+    import jinja2.ext  # noqa: F401
+    import numpy  # noqa: F401
```

Comment:
This library is not defined in setup.py? Should we add it? https://github.com/apache/airflow/blob/b104516/setup.py

Reply:
`numpy` is a direct dependency of `pandas` which we have in out setup.py

https://github.com/apache/airflow/blob/1faf985d83a15e0c2407e4d12a93437c299b44f2/setup.py#L719

https://github.com/pandas-dev/pandas/blob/4582c1c8701075a9f9f0bf75fbcd8546515b43e8/setup.py#L744-L750

Maybe a good idea to add it in `try .. catch` similar to the `kubernetes.client`

https://github.com/astronomer/airflow/blob/0b2906506bd39b8940787db145f0dffb2dd7f425/airflow/executors/celery_executor.py#L188-L191